### DISCUSSION TITLE
Segmentation Fault with Vehicles Breaks

### DISCUSSION THREAD

**Discussion by FreddyHunter** [google/or-tools#2991] [Routing (and legacy CP) questions]
Dear Community,

I am facing a Segmentation Fault error when adding vehicle breaks into my pyhton model. I just took the sample coding and add on my complete code, as you can see below. If I comment the breaks code part, it run again without any problem. Any tips or suggestion? Is someone facing it at latest version of OR Tools?

```python
    node_visit_transit = [0] * routing.Size()
    for index in range(routing.Size()):
        node = manager.IndexToNode(index)
        node_visit_transit[index] = ((data['demands'][node][0] + data['demands'][node][1] + data['demands'][node][2]) * data['time_per_demand'])
#        print(node_visit_transit[index])

    break_intervals = {}
    for vehicle_id in range(data['num_vehicles']):
        break_intervals[vehicle_id] = [
            routing.solver().FixedDurationIntervalVar(
                480,  # start min
                1080,  # start max
                60,  # duration: 10 min
                True,  # optional: no
                f'Break for vehicle {vehicle_id}')
        ]
        time_dimension.SetBreakIntervalsOfVehicle(
            break_intervals[vehicle_id],  # breaks
            vehicle_id,  # vehicle index
            node_visit_transit)
```

Thanks in advance for any sharing or thoughts.

**Comment by Mizux** [comment]
Instead of using `routing.Size()` you should try to use `manager.GetNumberOfIndices()` since the former return the number of indices but the number of end indices (which are always placed at the end of the solver internal indices array).

ref: https://github.com/google/or-tools/issues/2773#issuecomment-923765550
